src/components/contact/contactCTA/ContactCTA.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { FC } from "react";
+
+import {
+  DateSpan,
+  InfoContainer,
+  PanelsContainer,
+  ReachSpacer,
+  TimeSpan,
+  TitleText,
+  WhiteBGSectionLeft,
+  WhiteBGSectionRight,
+} from "./ContactCTA.css";
+
+const ContactCTA: FC = () => {
+  return (
+    <PanelsContainer id="panels-box">
+      <WhiteBGSectionLeft id="white-bg-section">
+        <TitleText>REACH US</TitleText>
```

Comment:
How would you feel about taking content in tags like `<TitleText>` and `<ReachSpacer>` amongst others, and placing it in a `Contact.onstants.ts` file? As we make more of these, we can refactor those constants up to `App.contstants.ts`

Reply:
I think that's a great idea. So, if I understand correctly, make some more generic components that value can be plugged into for different use cases?

Reply:
Not full fledged components, but a JS file to hold constants. It's similar to techniques used to handle internationalization, but we don't need to do it for multiple languages. Just put string constants into a separate file and import it into `Contact.tsx`